package.json

```diff
@@ -41,6 +41,7 @@
   "dependencies": {
     "@actions/core": "^1.10.0",
     "@actions/github": "^5.1.1",
+    "actions-toolkit": "github:nearform/actions-toolkit",
```

Comment:
we also need to remember that we need to move this to the new org once we move the toolkit (which I think makes sense moving along with the actions eventually). github may do the redirect, but for consistency we should reflect the name change in the references in all actions' repos. can you add this to the checklist please?